portal/static/js/gil.js

```diff
@@ -854,214 +955,425 @@ module.exports = OrgTool = (function() {
+                var la = l.attr("href");
+                if (l.length > 0 && window.app.utilObj.validateUrl(la)) {
+                  window.app.interventionSessionObj.clearSession(itemName);
+                  visObj.setRedirect();
+                  setTimeout(function() { location.replace(l.attr("href")); }, 0);
+                  return true;
                 };
-                __loader(false);
-              }).fail(function() {
-                __loader(false);
+              };
+            };
+          };
+          this.setItemVis = function(item, itemLink, vis) {
+            if (item) {
+              switch(vis) {
+                case "disabled":
+                  item.removeAttr("href");
+                  item.addClass("icon-box__button--disabled");
+                  break;
+                default:
+                  item.attr("href", itemLink);
+                  item.removeClass("icon-box__button--disabled");
+              }
+            }
+          };
+          this.handleInterventionItemLinks = function(interventionItem, customName){
+            var self = this;
+            if (interventionItem) {
+              var link = interventionItem.link_url;
+              var disabled = (String(link) === "disabled");
+              var itemName = customName||interventionItem.name;
+              var linkItems = $("." + itemName + "-link");
+              var menuItem = $("#intervention_item_" + itemName);
+              if (!disabled && menuItem.length === 0) { //only draw this when there isn't already one
+                $(".side-nav-items__item--dashboard").after('<li id="intervention_item_' + itemName + '" class="side-nav-items__item side-nav-items__item--has-icon side-nav-items__item--accentuated"><a href="' + link + '" class="capitalize intervention-link">' + interventionItem.description + '</a></li>');
+              }
+              if (linkItems.length > 0) {
+                if (!disabled) {
+                  linkItems.each(function() {
+                    self.setItemVis($(this), link);
+                  });
+                  $(".icon-box-" + itemName).removeClass("icon-box--theme-inactive");
+                } else {
+                  linkItems.each(function() {
+                    self.setItemVis($(this), link, "disabled");
+                  });
+                  $(".icon-box-" + itemName).addClass("icon-box--theme-inactive");
+                }
+              }
+            }
+          };
+          this.filterMenu = function (userId) {
+            if (!userId) {
+              return false;
+            }
+            var self = this;
+            var __PORTAL = $("#portalURI").val();
+            $.ajax({
+              url: __PORTAL + "/gil-interventions-items/" + userId,
+              context: document.body,
+              async: false,
+              cache: false
+            }).done(function(data) {
+              //console.log(data)
```

Comment:
Could you remove this, or have it only log if the log level is high enough (ie debugging on or development system)?